src/_main_/utils/emailer/send_email.py

```diff
@@ -18,14 +18,14 @@ def is_dev_env():
     return True
 
 
-def send_massenergize_email(subject, msg, to):
+def send_massenergize_email(subject, msg, to, sender):
```

Comment:
one occurance in message.py is missing this sender arg.  You could make it sender=None